docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1021 @@
+git clone git@github.com:myuser/logstash-{plugintype}-mypluginname.git
+----------------------------------
+
+==== Clone and copy the example plugin (highly recommended)
+At this point, you could build the plugin out entirely on your own.  Plugins
+need to have a similar directory structure as other ruby gems.
+
+[NOTE]
+=========================
+This example ruby gem file structure
+(and excellent walkthrough of ruby gem creation) is from
+http://timelessrepo.com/making-ruby-gems :
+[source,txt]
+[subs="attributes"]
+----------------------------------
+$ tree mygem
+├── lib/
+│   ├── mygem.rb
+│   └── mygem/
+│       ├── base.rb
+│       └── version.rb
+├── test/
+├── bin/
+├── Rakefile
+└── mygem.gemspec
+----------------------------------
+=========================
+
+For Logstash plugins, your file structure should look like this:
+
+[source,txt]
+[subs="attributes"]
+----------------------------------
+$ tree logstash-{plugintype}-mypluginname
+├── Gemfile
+├── LICENSE
+├── README.md
+├── Rakefile
+├── lib
+│   └── logstash
+│       └── {plugintype}s
+│           └── mypluginname.rb
+├── logstash-{plugintype}-mypluginname.gemspec
+└── spec
+    └── {plugintype}s
+        └── mypluginname_spec.rb
+----------------------------------
+
+
+[TIP]
+=========================
+Here's a relatively easy way to import the necessary files and
+directory structure.
+
+[source,js]
+[subs="attributes"]
+----------------------------------
+cd /tmp
+git clone git@github.com:logstash-plugins/logstash-{plugintype}-{pluginname}.git
+cd logstash-{plugintype}-{pluginname}
+rm -rf .git <1>
+cp -R * /path/to/logstash-{plugintype}-mypluginname/
+<2>
+cd /path/to/logstash-{plugintype}-mypluginname
+mv logstash-{plugintype}-{pluginname}.gemspec logstash-{plugintype}-mypluginname.gemspec <3>
+mv lib/logstash/{plugintype}s/{pluginname}.rb lib/logstash/{plugintype}s/mypluginname.rb <4>
+mv spec/{plugintype}s/{pluginname}_spec.rb spec/{plugintype}s/mypluginname_spec.rb <5>
+----------------------------------
+
+<1> We don't need this `.git` directory or its contents.  They match the example
+repository instead of our newly created repository, so we will delete them here.
+Don't forget to clean up the temporary directory after you've copied the
+contents. Your file structure should look like the Logstash plugin example
+shown.
+
+<2> Since we've copied the files from the logstash-{plugintype}-{pluginname}
+repository, there are a few files we should rename accordingly.
+
+<3> The gemspec file needs to be renamed.
+<4> Inside the `lib/logstash/pass:attributes[{plugintype}]s/` directory is the
+`pass:attributes[{pluginname}].rb` file.  It should be renamed to match
+`mypluginname`, e.g.
+
+<5> In like fashion, inside the `spec/pass:attributes[{plugintype}]s/` directory is
+the `pass:attributes[{pluginname}]_spec.rb` file.  It, too, should be renamed to
+match `mypluginname`, e.g.
+=========================
+
+=== An overview of your plugin
```

Comment:
So it fits with the other headings, I'd call this "See what your plugin looks like" For the intro text, maybe:

Before we dive into the details, open up the plugin file in your favorite text editor
and take a look. 

<example>

I don't think it's necessary to show the vi command, just the file contents should be sufficient.